.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 dist
-package-lock.json
```

Comment:
Seems as non-related change.

Reply:
Yes, but i'm lazy to open another PR for that.